### PR TITLE
Refactor/build setup

### PR DESCRIPTION
**Pull request by bor3ham**
Awkward - I published this as v0.3.1 before knowing master was write locked

Also I can't explain the version jump - Yarn told me the current was v0.3.0 :|

### CODE REVIEW

**Comment by bor3ham** [comment]
Okay I've purged that travesty from NPM, carry on

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 node_modules/
-dist/
+dist/*.css
```
Comment (scaredcat) [inline .gitignore]: ??

Comment (scaredcat) [reply .gitignore]: please do it properly and use a `.npmignore` file

Comment (bor3ham) [reply .gitignore]: Not to do with npm packaging magic - you guys just weren't using anything in `dist` so I made it more specific when I started using it for the css build

Comment (scaredcat) [reply .gitignore]: I see. I still think we don't need to upload anything in the dist folder.

**Pull request merged**